src/consul_configs.rs

```diff
@@ -48,14 +49,14 @@ pub fn create_contract_abi(
 
 fn consul_select<F>(
     keyname: &str,
-    consul_uri: &str,
+    consul_url: &str,
```

Comment:
Compelling reason for this change?

Reply:
For consistency, I changed here because the other functions are using `consul_url`.